modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -306,10 +321,37 @@ To configure an external load balancer:
 . Review the information described in *Advanced Options* when adding an external load balancer.
 . Configure DNS before using the CN obtained from the TLS certificate. DNS is required to send requests to applications or API gateways deployed to Runtime Fabric. Add an "A record" to your DNS provider to map the CN to the IP address of the external load balancer or controller VMs.
 
-== Step 5: Deploy Applications
-When you are ready to deploy an application, follow the instructions in xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric].
+== Step 4: Deploy Applications
```

Comment:
```suggestion
== Deploy Applications
```
Always introduce subtasks in the intro to a supertask, so that we don't have to identify subtasks as steps....